sphinx_gallery/tests/test_utils.py

```diff
@@ -10,10 +10,11 @@
 
 from __future__ import division, absolute_import, print_function
 import sphinx_gallery.utils as utils
+from sphinx_gallery.tests.test_gen_gallery import sphinx_app_wrapper
```

Comment:
Ignorance speaking. Why do you need this here?

Reply:
was a remnant from an earlier part of this PR - removing in next push